features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/wso2/templates/application/application-edit/js/application-edit.js

```diff
@@ -13,7 +13,7 @@ $(document).ready(function () {
 
      $.validator.addMethod('validateSpecialChars', function(value, element) {
         return /(^[a-zA-Z0-9 ._-]*$)/g.test(value);
-     }, i18n.t("The name contains one or more illegal characters") + '( &nbsp;&nbsp; " &nbsp;&nbsp; \' &nbsp;&nbsp; )');
+     }, i18n.t("The name contains one or more illegal characters") + '(~ ! @ #  ; % ^ & * + = { } | &lt; &gt;, \' " \\ \/ )');
```

Comment:
Isn't it better to say allowed only string and numbers rather listing all illegal characters. Just a suggestion.